Examples/Detectors/GenericDetector/src/BuildGenericDetector.cpp

```diff
@@ -11,8 +11,680 @@
 #include <cmath>
 #include <numbers>
 
+namespace ActsExamples::Generic {
+
+std::unique_ptr<const Acts::TrackingGeometry> buildDetector(
```

Comment:
```suggestion
std::unique_ptr<const Acts::TrackingGeometry> Generic::buildDetector(
```
guards potential future linker errors

Reply:
Ok. I remember you mentioned this before, but I've never seen it happen.